parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -241,18 +241,6 @@ loop = do
       getHQ'Terms p = BranchUtil.getTerm (resolveSplit' p) root0
       getHQ'Types :: Path.HQSplit' -> Set Reference
       getHQ'Types p = BranchUtil.getType (resolveSplit' p) root0
-      getHQTerms :: HQ.HashQualified Name -> Action' m v (Set Referent)
```

Comment:
This was moved to the top-level in an effort to start breaking up this enormous function :)